static/css/event.less

```diff
@@ -0,0 +1,210 @@
+/* ==========================================================================
+   cfgov-refresh
+   Page-specific styles for event post detail page.
+   ========================================================================== */
+
+.post-preview__event {
+    border-top: 1px solid @gray-50;
+    padding-top:  1em;
+
+    .media-object_image-container {
+         .respond-to-max(@mobile-max {
+            display: none;
+        });
+    }
+}
+
+.summary {
```

Comment:
I feel like this is targeting too high, but if we're keeping it, these should be moved in w/ the rest of the [summary styles](https://github.com/cfpb/cfgov-refresh/blob/33a5e3a2b62b451f3403785e5b66371cc42d47d5/static/css/summary.less)